File: contractstudy/src/test/java/test/contractstudy/extractor/common/StaticImportCollector/java/testData/StaticImportSomeStatic.java
package test.contractstudy.extractor.common.StaticImportCollector.java.testData;

import java.util.List;

import static java.lang.System.out;

public class StaticImportSomeStatic {

  public boolean test(boolean flag) {
    return !flag;
  }

  public double sum(int num1, double num2) {
    return num1 + num2;
  }

  public long getTimeStamp(int area1) {
    return System.currentTimeMillis();
  }

  public void print(List<String> list) {
    for (String item : list) {
      out.println(item);
    }
  }
}

File: contractstudy/src/test/java/contractstudy/collectContracts/common/StaticImportCollector/kotlin/testData/StaticImportSomeStatic.kt
package contractstudy.collectContracts.common.StaticImportCollector.kotlin.testData

import com.google.common.base.Preconditions


class StaticImportSomeStatic {
    fun test(flag: Boolean): Boolean {
        Preconditions.checkArgument(flag, "This is an error")
        return !flag
    }

    private fun sum(num1: Int, num2: Double): Double {
        return num1 + num2
    }

    fun compute(area1: Int): Double {
        return sum(area1, Math.PI)
    }

    fun print(text: String?) {
        println(text)
    }
}